packages/venia-concept/jsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What be this file? If it's specific to VSCode did you mean to `gitignore` it?

Reply:
Not sure what this is but it should be ignored.

Reply:
Ignoring it. I do recommend you drop a file like this in your venia-concept folder, because it helps VSCode to understand and autocomplete your import statement paths from `src/`.

Reply:
Yea I'm totally on board as a VSCode user, just didn't know if we wanted editor configs in the repo.